modules/tinymce/src/core/test/ts/browser/keyboard/SpaceKeyTest.ts

```diff
@@ -32,7 +39,7 @@ describe('browser.tinymce.core.keyboard.SpaceKeyTest', () => {
       editor.nodeChanged();
       TinyContentActions.keystroke(editor, Keys.space());
       TinyAssertions.assertSelection(editor, [ 0, 1, 0 ], 2, [ 0, 1, 0 ], 2);
-      TinyAssertions.assertContent(editor, '<p>a <a href="#">b&nbsp;</a> c</p>');
+      TinyAssertions.assertContent(editor, '<p>a <a href="#">b </a> c</p>');
```

Comment:
This change is not correct and something you've done has introduced a regression/bug as this should not have changed.

Reply:
Are you sure this is not correct? What's happening with this is that we are triggering inputkeys listener which runs `normalizeNbspsInEditor`. If it isn't correct, do we have a better way than ignoring events with `data: ' '`?

Reply:
100% sure this is incorrect. Take what you've said is the new expected and dump it into a browser. You'll see there's only 1 space after `b` because the browser will collapse adjacent spaces so the original was correct here.

In this case we either have a bug in the `InputKeys` override logic or we don't want the override to run since we've already normalised it. I'm not sure which is the issue so you'll need to investigate that more. If it is the latter, then one way to workaround it would be to add a custom property to the fake events and update the override to ignore the event in that case. We should only do that if we're sure there's not a bug in the overrides though.